packages/ui/src/components/Input/Input.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { Input } from './index';
```

Comment:
To reduce the chance for import cycles you can import Input from the `./Input` file